### ISSUE TITLE
Need to dismiss the dropdown on a window click

### ISSUE THREAD

**Issue by sglanzer** [ciena-frost/ember-frost-select#22]
Clicking anywhere except for the dropdown should dismiss the dropdown

**Comment by cstolli** [comment]
this is trickier than it sounds, I would propose we make this responsibility of the consuming context and not the drop-down itself.

otherwise, "blur" is about the only way to handle it, and that could mean a lot of things in a complex component like this.  open to suggestions.

**Comment by sglanzer** [comment]
Let's chat on Slack, I did something like this for the popovers

**Comment by job13er** [comment]
Agreed that while tricky, this is worth investigating. UX as it is currently is frustrating as there doesn't appear immediately to be a way to dismiss the dropdown. @sandersky eventually told me I could click the down arrow, I would never have tried that.

**Comment by cstolli** [comment]
@job13er the `esc` key always works as well
